tests/test_web/test_api_v1.py

```diff
@@ -46,6 +46,7 @@ def test_get_build(app, auth_env, client, db):
         'binary_image_resolved': 'quay.io/namespace/binary_image@sha256:abcdef',
         'bundle_mapping': {},
         'bundles': ['quay.io/namespace/bundle:1.0-3'],
+        'deprecation_list': [],
```

Comment:
I have a feeling that this test is not complete. If you use `get_build` function. You can get more request types that don't have the same structure of the response. Would it be useful to modify the name of the test that we know it tests only add request_type?

Reply:
I think this test was written more to test the outcome of the `get_build` endpoint rather than specific response types. The other request types have their own tests. But these are my two cents. Please let me know what you think

Reply:
Sounds ok if the intent was this.